.changeset/reduced_accepted_values_for_js_options.md

```diff
@@ -7,3 +7,4 @@ biome_js_formatter: major
 - The type `Semicolon` can't be converted from `"AsNeeded"` and `"Always"`.
 - The type `TrailingCommas` can't be converted from `"ES5"`, `"All"` and `"None"`.
 - The type `ArrowParenthesis` can't be converted from `"AsNeeded"` and `"Always"`.
+- The type `JsxRuntime` can't be converted from `react-classic`.
```

Comment:
The changeset needs to be corrected. The change was done to `biome_configuration`, not `biome_js_formatter`.

Since `biome_configuration` is a "private" crate, we can avoid a new changeset.